src/main/java/org/opentripplanner/model/plan/Leg.java

```diff
@@ -244,6 +245,10 @@ default boolean getRealTime() {
     return false;
   }
 
+  default RealTimeState getRealTimeState() {
+    return RealTimeState.SCHEDULED;
```

Comment:
I don't know if it's better to return SCHEDULED or null? SCHEDULED for street legs feels a bit wrong, on the other hand that's how it worked in otp1 seemingly.

Reply:
I would probably go with null.

Reply:
Changed.